plugins/GSdx/GSRendererHW.h

```diff
@@ -29,7 +29,7 @@
 
 class GSRendererHW : public GSRenderer
 {
-private:
+protected:
```

Comment:
I won't say a lot. Only a couple of stuff. A function might be a good idea, it could be possible to use it at difference place.